daemon/mgr/container.go

```diff
@@ -1424,6 +1432,66 @@ func (mgr *ContainerManager) parseBinds(ctx context.Context, meta *ContainerMeta
 	return nil
 }
 
+func (mgr *ContainerManager) setMountPointDiskQuota(ctx context.Context, c *ContainerMeta) error {
+	type diskQuotaRegExp struct {
```

Comment:
I do not think defining a struct in function `setMountPointDiskQuota ` is a good idea.
Maybe we could  move this into diskquota package. WDYT? @rudyfly